quantum/quantum_keycodes.h

```diff
@@ -71,6 +71,12 @@ enum quantum_keycodes {
     QK_TAP_DANCE_MAX      = 0x57FF,
     QK_LAYER_TAP_TOGGLE   = 0x5800,
     QK_LAYER_TAP_TOGGLE_MAX = 0x58FF,
+#ifdef STENO_ENABLE
```

Comment:
Would you mind moving these? [There's a PR open right now that's occupying that space](https://github.com/qmk/qmk_firmware/pull/1514/files#diff-438068f4a25ff76f3e13720fd8436756R70) :)